sygnal/helper/proxy/proxyagent_twisted.py

```diff
@@ -89,7 +95,13 @@ def __init__(
         self._policy_for_https = contextFactory
         self._reactor = reactor
 
-    def request(self, method, uri, headers=None, bodyProducer=None):
+    def request(
+        self,
+        method: bytes,
+        uri: bytes,
+        headers: Optional[Headers] = None,
+        bodyProducer: Optional[IBodyProducer] = None,
+    ) -> "defer.Deferred[IResponse]":
```

Comment:
Could you remove the types from the docstring? We generally prefer to have the types specified in the signature only, to prevent the types in docstring from going out of sync.

Reply:
Done